api/app/signals/apps/api/v1/views/questions.py

```diff
@@ -0,0 +1,27 @@
+from datapunt_api.rest import DatapuntViewSet, HALPagination
+from django.db.models import Q
+
+from signals.apps.api.v1.serializers import PublicQuestionSerializerDetail
+from signals.apps.signals.models import Question
+
+
+class PublicQuestionViewSet(DatapuntViewSet):
+    queryset = Question.objects.all()
+
+    serializer_class = PublicQuestionSerializerDetail
+    serializer_detail_class = PublicQuestionSerializerDetail
+    pagination_class = HALPagination
+    view_name = 'question-detail'
+
+    def get_queryset(self, *args, **kwargs):
```

Comment:
In general we loose the `order` of the questions because there is no ordering on the QuerySet. So add an `order_by` ordering on the `CategoryQuestion.order`

Reply:
check